UI/GalaxySetupWnd.cpp

```diff
@@ -726,6 +726,9 @@ void GalaxySetupPanel::CompleteConstruction() {
     m_monster_freq_list->Insert(GG::Wnd::Create<CUISimpleDropDownListRow>(UserString("GSETUP_MEDIUM")));
     m_monster_freq_list->Insert(GG::Wnd::Create<CUISimpleDropDownListRow>(UserString("GSETUP_HIGH")));
     m_monster_freq_list->Insert(GG::Wnd::Create<CUISimpleDropDownListRow>(UserString("GSETUP_RANDOM")));
+    m_monster_freq_list->Insert(GG::Wnd::Create<CUISimpleDropDownListRow>(UserString("GSETUP_MICROSCOPIC")));
+    m_monster_freq_list->Insert(GG::Wnd::Create<CUISimpleDropDownListRow>(UserString("GSETUP_TINY")));
+    m_monster_freq_list->Insert(GG::Wnd::Create<CUISimpleDropDownListRow>(UserString("GSETUP_VERY_LOW")));
```

Comment:
due to how only the selected index from that dropdownlist gets transmitted through server/savegame/universecreation, i had to put the new enum values to the end of the list. this will be prettier in the next iteration but is fully working in this ugly form already for testing purposes.

Reply:
TINY and MICROSCOPIC don't fit in with the current naming if you ask me. I'd go for something like VERY_HIGH, HIGH, MEDIUM, LOW, VERY_LOW and dismiss MICROSKOPIC.

Reply:
how about this skala?

0.002 extremely low
0.01 very low
0.03 low
0.08 medium
0.2 high
0.5 very high
0.99 extremely hgh

Reply:
and done.